lua/astrocommunity/git/mini-git/init.lua

```diff
@@ -0,0 +1,8 @@
+---@type LazySpec
+return {
+  "echasnovski/mini-git",
+  event = "User AstroFile",
+  cmd = { "Git" },
+  version = false,
+  main = "mini.git",
+}
```

Comment:
```suggestion
  opts = {},
}
```

might as well put this since there is a `setup` function